playbooks/roles/wordpress/files/sites/default.opencraft.com.conf

```diff
@@ -0,0 +1,31 @@
+server {
```

Comment:
@Agrendalath, all the host-specific configuration should be moved to the `ansible-secrets` repository in a file under the `host_vars` directory and only the generic playbook files and templates should be in this repository.

Reply:
Done.